src/eppo-client.ts

```diff
@@ -1,4 +1,4 @@
-import { createHash } from 'crypto';
+import * as md5 from 'md5';
```

Comment:
The `crypto` library is native to Node but is not supported in browser `<script>` tags. Therefore, I had to use a substitute md5 hash library.